pkg/skaffold/event/v2/logger.go

```diff
@@ -40,7 +43,7 @@ func (l logger) Write(p []byte) (int, error) {
 	handler.handleSkaffoldLogEvent(&proto.SkaffoldLogEvent{
 		TaskId:    fmt.Sprintf("%s-%d", l.Phase, handler.iteration),
 		SubtaskId: l.SubtaskID,
-		Level:     0,
+		Level:     -1,
```

Comment:
what does the level `-1` represent?

Reply:
this is basically just to avoid marking normal output with a level, since they technically don't have one as they don't come from any logging library. I wasn't sure how to best represent this. I can put it in a variable to make things clear or leave a comment to clarify if you think that's best

Reply:
just a comment should suffice